boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -53,6 +59,24 @@ class FsmStateMachineTest : public testing::Test
         RESET_FAKE(send_non_periodic_msg_FSM_WATCHDOG_TIMEOUT);
         RESET_FAKE(get_current_ms);
         RESET_FAKE(heartbeat_timeout_callback);
+        RESET_FAKE(get_primary_flow_rate);
+        RESET_FAKE(get_secondary_flow_rate);
+    }
+
+    void SetInitialState(const struct State *const initial_state)
+    {
+        App_SharedStateMachine_Destroy(state_machine);
+        state_machine = App_SharedStateMachine_Create(world, initial_state);
+        ASSERT_TRUE(state_machine != NULL);
+        ASSERT_EQ(
+            initial_state,
+            App_SharedStateMachine_GetCurrentState(state_machine));
+    }
+
+    std::vector<const struct State *> GetAllStates(void)
+    {
+        return std::vector<const struct State *>{ App_GetAirOpenState(),
+                                                  App_GetAirClosedState() };
```

Comment:
The convention thus far is to put Setup and Teardown at the top, then all other functions below them

Reply:
should be fixed now